x-pack/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/confirm_delete.js

```diff
@@ -5,40 +5,42 @@
  */
 
 import React, { Component } from 'react';
-import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
 import { EuiOverlayMask, EuiConfirmModal } from '@elastic/eui';
 import { toastNotifications } from 'ui/notify';
 import { deletePolicy } from '../../../../services/api';
 import { showApiError } from '../../../../services/api_errors';
-export class ConfirmDeleteUi extends Component {
+
+export class ConfirmDelete extends Component {
```

Comment:
TODO: Confirm this looks correct in the UI.

Reply:
![image](https://user-images.githubusercontent.com/1238659/59351990-75462f00-8cd4-11e9-8baf-51f3f5a3709f.png)